src/utils/componentUtilityFunctions.js

```diff
@@ -312,7 +312,7 @@ export const filterValidAttributes = props =>
 export const browserSupports = api => {
   switch (api) {
     case 'ResizeObserver':
-      return typeof ResizeObserver !== 'undefined';
+      return typeof window.ResizeObserver !== 'undefined';
```

Comment:
Should we nullcheck `window` in the event it's not available in non-browser environments?

```suggestion
      return typeof window?.ResizeObserver !== 'undefined';
```